Sources/SwiftInspectorVisitors/PropertyVisitor.swift

```diff
@@ -158,15 +158,25 @@ public final class PropertyVisitor: SyntaxVisitor {
   }
 
   private func findInitializerDescription(from node: PatternBindingListSyntax) -> String? {
-    let initializerClauseSyntaxes = node.compactMap { $0.initializer }
+    let initializerClauseSyntaxes = node.compactMap { $0.initializer?.withEqual(nil) }
     assert(initializerClauseSyntaxes.count <= 1, "A property should have at most one initializer.")
     return initializerClauseSyntaxes.first?.description
   }
 
   private func findCodeBlockDescription(from node: PatternBindingListSyntax) -> String? {
-    let accessors = node.compactMap { $0.accessor }
-    assert(accessors.count <= 1, "A property should have at most one accessor.")
-    return accessors.first?.description
+    guard
+      let patternBindingSyntax = node.children.first?.as(PatternBindingSyntax.self),
+      let codeBlockSyntax = patternBindingSyntax.children.compactMap({ $0.as(CodeBlockSyntax.self) }).first
```

Comment:
If we wanted we could assert here that there are only one of these nodes, though I feel like that's optional. We may want to come up with a general philosophy of when we think assertions are necessary when parsing nodes like this.